Makefile

```diff
@@ -1,7 +1,15 @@
-testpop:	clean
-	godep go build scripts/testpop/testpop.go
+.PHONY: clean testpop mocks out
+
+GOPATH = $(shell echo $$GOPATH)
+
+out:	testpop
 
 clean:
 	rm -f testpop
 
-.PHONY: clean testpop
+mocks:
+	mockery -dir $(GOPATH)/src/github.com/uber-common/bark -name Logger -output testutils/mocks/ -recursive
```

Comment:
This generates a file like `Logger.go` and `StatsReporter.go` (with captial letters). Is this required? Can we make the generated files more lowercase-y.